harpy/common/workflow.py

```
@@ -0,0 +1,289 @@
+             print_error(
+                 "snakefile missing",
+                 f"The required snakefile [blue bold]{self.snakefile}[/] was not found in the Harpy installation."
+             )
+             print_solution(
+                 "There may be an issue with your Harpy installation, which would require "
+                 "reinstalling Harpy. Alternatively, there may be an issue with your "
+                 "conda/mamba environment or configuration."
+             )
+             sys.exit(1)
+
+    def fetch_snakefile(self):
```

Ruff (0.12.2)

Redefinition of unused `fetch_snakefile` from line 164

(F811)